java/daikon/DynamicConstants.java

```diff
@@ -189,19 +190,19 @@ public Constant (VarInfo vi) {
 
 
     /*@EnsuresNonNullIf(result=true, expression="#1")*/
-    /*@Pure*/ public boolean equals (/*@Nullable*/ Object obj) {
+    /*@Pure*/ public boolean equals (/*>>>@GuardSatisfied Constant this,*/ /*>>>@GuardSatisfied @Nullable*/ Object obj) {
```

Comment:
Use /*...*/ style comments whenever possible.  In particular, use two of them rather than one voodoo-style comment /*>>>...*/.